web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -658,10 +713,83 @@ const TeamReviews = ({ onBack, periodId }) => {
     }
   };
 
+  const approveAll = () => {
+    visibleTeamMembers().map(member => approveMember(member, true));
+  };
+
+  const unapproveAll = () => {
+    visibleTeamMembers().map(member => approveMember(member, false));
+  };
+
+  const toggleApproval = async member => {
```

Comment:
This toggles the approval state of the assignments for all reviewers of a given member.